src/components/Collective.vue

```diff
@@ -94,6 +97,20 @@ export default {
 		},
 		'currentPage.id'() {
 			this.selectVersion(null)
+
+			const routerParams = this.$router.currentRoute.params
+			// If the current page is not the one we are supposed to be on, redirect
```

Comment:
I have to admit that I don't fully understand the purpose of this code. When is this expected to happen? Is the idea to redirect to the slugified URL when the old URL is opened?

Reply:
yeah, exactly. The only one thing: it stopped work after one of latest rebases. I'm investigating it